pkg/runtime/wfengine/workflow.go

```diff
@@ -62,6 +62,11 @@ type recoverableError struct {
 	cause error
 }
 
+type PolicyAndEventData struct {
+	Policy    *api.OrchestrationIdReusePolicy `json:"policy"`
+	EventData []byte                          `json:"eventData"`
```

Comment:
For consistency with existing code, I suggest `StartEventData`.
```suggestion
	StartEventBytes []byte                    `json:"startEventBytes"`
```